File: Program12.c
// To Check whether the Number is Divisible by 3 and 5 

#include<stdio.h>
#include<stdbool.h>

bool DivisibleByThreeAndFive(int iNum)
{
    int iAns1 = 0,iAns2 = 0 ;

    iAns1 = iNum % 3 ;
    iAns1 = iNum % 5 ;

    if((iAns1 == 0 ) && (iAns2 == 0))
    {
        return true ;
    }
    else 
    {
        return false ;
    }
}

int main()
{
    int iValue = 0 ;
    bool bResult = false ;

    printf("Enter the Number \n");
    scanf("%d",&iValue);

    bResult = DivisibleByThreeAndFive(iValue);

    if(bResult == true )
    {
        printf("%d is divisible by 3 and 5 \n",iValue);
    }
    else
    {
        printf("%d is not Divisible by 3 or 5\n",iValue);
    }

    return 0 ;


}

File: Program12.py
# Accept Number from user and check Whether that number is perfect or not 
# Input : 6 
# Output : Number is perfect Number 

def checkperfect(iValue):
    iSum = 0 
    for i in range(1,iValue):
        if(iValue % i == 0 ):
            iSum = iSum + i 
    
    if(iSum == iValue ):
        return True  
    else : 
        return False 

def main():
    iNo = 0  
    bRet = False 
    
    iNo = int(input("Enter a number : "))

    bRet = checkperfect(iNo)

    if(bRet == True ):
        print("Number is perfect")
    else : 
        print("Number is not perfect")

if __name__=="__main__":
    main()